### ISSUE TITLE
Weird boxing size for label

### ISSUE THREAD

**Issue by acivita** [amcharts/amcharts4#4129]
hello :).

I hope you are all doing fine.

I am trying to use label.html to get a custom label.

I have an issue about the width of the container (is not 100%, its less or more).

To check if the container was the problem, i put some red background.
![capture stack overflow](https://user-images.githubusercontent.com/8580643/185111594-62818456-f34a-434c-9311-439b939dc0cc.PNG)

On this example, the text is not properly center because horizontalCenter is based on the container and not the actual width.

I tried to reproduce the code in codepen, but this problems doesn't seem to happen (I can't see the red background).

Both code are identicals.

There is my code pen : https://codepen.io/acivita/pen/vYRmzgO

Verticaly align my items is not working as well but i try to understand the first issue.

Thank you for your help

**Comment by acivita** [comment]
So i tried a few things with codepen : 

First, i make a html/css who is exactly what i would like :

**Comment by martynasma** [comment]
Your second CodePen looks good, except the label gets zoomed in together with the map.

You can just disable the scaling:

```JavaScript
label.nonScaling = true;
```

**Comment by acivita** [comment]
Hello, how are you ?

Thank you for your answer.

I've already tried the nonScaling property, it works good but as it doesn't scale if I zoom out, it keeps the same size. then its too big.

For example ; https://codepen.io/acivita/pen/KKobeXm

I set the good size at a zoom level of 15, but if the zoom changes it doesn't scale (normal because of nonScaling) and ruins my map because of the cities orverlap.

I really want something that doesn't scale with a very small size where properties like horizontal center works.

I may try to hide them at the a certain zoomLevel but i'm not sure it's possible (but i found it a bit ugly).

Thank for your help and have a nice day.
